test/utils/expected-files.js

```diff
@@ -660,6 +660,7 @@ const expectedFiles = {
     cucumber: [
         `${TEST_DIR}features/user/user.feature`,
         `${TEST_DIR}features/gitkeep`,
+        `${SERVER_TEST_SRC_DIR}com/mycompany/myapp/cucumber/stepdefs/CucumberContextConfiguration.java`,
```

Comment:
[Tests are failing](https://travis-ci.org/jhipster/generator-jhipster/jobs/479624834) because you forgot to update this line.
It should be the last required change, and then we should be able to merge it ;-)